spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -27,6 +28,18 @@
   describe '#index' do
     subject(:action) { get :index, params: params }
 
+    context 'with prompt=login' do
+      let(:prompt) { 'login' }
+
+      it 'does not log user out when switching languages after authentication' do
+        user = create(:user, :with_phone)
+        action
+        sign_in_as_user(user)
+        get :index, params: params.merge(locale: 'es')
+        expect(controller.current_user).to_not be_nil
```

Comment:
This would fail prior to the changes implemented here.

Reply:
```suggestion
        expect(controller.current_user).to eq(user)
```
Should we check the specific user?

Reply:
Oops, yep, thank you